ui-components/src/actions/Button.tsx

```diff
@@ -51,6 +52,7 @@ export const buttonInner = (props: Omit<ButtonProps, "onClick">) => {
           size={props.iconSize ?? iconSize}
           symbol={props.icon}
           iconClass={props.passToIconClass}
+          fill={props.iconColor ?? IconFillColors.primary}
```

Comment:
Needed to default these to the primary color (this was existing, but affected the button)